src/bsonjs.c

```diff
@@ -42,8 +43,20 @@ bson_str_to_json(const char *bson, size_t bson_len, size_t *json_len)
         bson_reader_destroy(reader);
         return NULL;
     }
-
-    json = bson_as_json(b, json_len);
+    if(strcmp(extended_str, "relaxed") == 0){
+        json = bson_as_relaxed_extended_json(b, json_len);
+    }
+    else if(strcmp(extended_str, "canonical") == 0){
+        json = bson_as_canonical_extended_json(b, json_len);
+    }
+    else if(strcmp(extended_str, "legacy") == 0){
+        json = bson_as_json(b, json_len);
+    }
+    else{
+        PyErr_SetString(PyExc_ValueError, "The value of extended must be one\n"
```

Comment:
Is there supposed to be a "\n" in this error message?

Reply:
Done.